### ISSUE TITLE
No diagram or editor are shown

### ISSUE THREAD

**Issue by rawIce** [peterfromearth/dokuwiki-plugin-bpmnioeditor#1]
Hi Peter, 

we have tried to use bpmnioeditor plugin to embed process definitions inside DokuWiki, however it does not show anything visible to the user.

There seems to be HTML code loaded to the page, however all elements seems to be hidden or invisble via CSS.  

I have tested the following code inside DokuWiki: 

1. link to an existing file including namespace
2. link to an existing file without namespace
3. link to an non existing file

```
==== Test of bpmnioeditor ====

{{playground:diagram.bpmn}}

{{diagram.bpmn}}

{{idonotexist.bpmn}}

```

This is how it looks like for the user: 
![image](https://user-images.githubusercontent.com/42028712/153023819-790b4c55-bf5f-48f5-9e3a-1db80acc2d73.png)

What I have already checked: 
- bpmn definition in mime.local.conf
- no errors in JavaScript console in browser 
- no difference if adding namespace to file name

Environment: 

- Browser: Microsoft Edge Version 98.0.1108.43 (Offizielles Build) (64-Bit)
- DokuWiki Version: Release 2020-07-29 "Hogfather"
- PHP Version: 7.4.3
- OS: Ubuntu 20.04.3 LTS

**Comment by peterfromearth** [comment]
to be sure that its not an caching issue, can you call <website>/lib/exe/js.php?purge and <website>/lib/exe/css.php?purge
Reload the page with STRG+F5

otherwise:
- how did you create the bpmn files?
- can you look at the network tab in the browsers developer tools, maybe the some calls have a problem

**Comment by rawIce** [comment]
- called /lib/exe/js.php?purge and /lib/exe/css.php?purge : done
- additionally invalidated cache by saving the configuration from the [Configuration Manager] as suggested here https://www.dokuwiki.org/caching : done 

- how did you create the bpmn files?
I tested two, the easier one was the default file which I can get from https://demo.bpmn.io/s/start by just opening the link and saving the default diagram

- can you look at the network tab in the browsers developer tools, maybe the some calls have a problem
All fine, there has always been an issue with some dokuwiki.svg manifest, but this has always been the case and also happens on the official https://www.dokuwiki.org/dokuwiki page

**Comment by jfburdet** [comment]
Jumping on the thread : I have the same problem : DokuWiki using docker image ghcr.io/linuxserver/dokuwiki:version-2020-07-29

**Comment by peterfromearth** [comment]
sorry for the long delay

can you try again with version 2022-02-22?

if it does not work please reopen the issue.

Peter

**Comment by rawIce** [comment]
I updated to 2022-02-22 using plugin manager, deleted browser cache, resaved DokuWiki Config file to invalidate cache and also called /lib/exe/js.php?purge and /lib/exe/css.php?purge - however no visual difference. 

In source code it says now `style="display:none"` so I think the update has worked, however there is still nothing rendered. 
Again networking and console show no errors. 

If i manually remove the `style="display:none"` property of the parent `<div id="bpmn_canvas_1_view"` it shows an empty white box.  There is also no `class="bjs-container"` present. 

I played around with the page and noticed if I make the `<button id="bpmn_1_view_button"` visible and press it, the page reloads and the module is indeed visible and the bpmn.io frame loads! 

So I assume there is still something wrong while initializing the page or in the hidden/visible logic.
